bzr.go

```diff
@@ -94,6 +95,20 @@ func (s *BzrRepo) Version() (string, error) {
 	return strings.TrimSpace(string(out)), nil
 }
 
+// Date retrieves last commit date.
+func (s *BzrRepo) Date() (time.Time, error) {
+	out, err := s.runFromDir("bzr", "version-info", "--custom", "--template={date}")
+	if err != nil {
+		return time.Time{}, err
+	}
+	const longForm = "2006-01-02 15:04:05 -0700"
```

Comment:
It appears the `const longForm` is used in each implementation. Can the definition be moved to the `repo.go` file to avoid duplication.